packages/autocomplete-js/src/autocomplete.ts

```diff
@@ -52,6 +52,11 @@ export function autocomplete<TItem extends BaseItem>(
         optionsRef.current.shouldPanelShow ||
         (({ state }) => {
           const hasItems = getItemsCount(state) > 0;
+
+          if (!props.value.core.openOnFocus) {
```

Comment:
We need to check for `!state.query` as well otherwise the `empty` template will never display when `openOnFocus: false`.

Reply:
Good catch, I've updated it with a test https://github.com/algolia/autocomplete.js/pull/407/commits/e9ec1d6ba3d8675948509d9356c1bf0ad9b22365